main.js

```diff
@@ -160,4 +160,151 @@ const inventory = [
     originalStock: 10,
     sold: 8,
   },
-];
\ No newline at end of file
+];
+
+// OPDRACHT 1
+//opdracht 1a
+const televisionTypes = inventory.map((television) => {
+  return television.type;
+})
+console.log(televisionTypes);
+
+//opdracht 1b
+const soldOut = inventory.filter((television) =>{
+  if (television.originalStock - television.sold === 0){
```

Comment:
Je zou hier een iets simpelere check van kunnen maken
```suggestion
  if (television.originalStock === television.sold){
```